src/node_crypto.cc

```diff
@@ -309,9 +288,65 @@ void ThrowCryptoError(Environment* env,
     // 'error:0906700D:PEM routines:PEM_ASN1_read_bio:ASN1 lib',
     // 'error:0D07803A:asn1 encoding routines:ASN1_ITEM_EX_D2I:nested asn1 err'
     // ]
-    exception->Set(env->context(), env->openssl_error_stack(), error_stack)
+    context->Set(env->context(), env->openssl_error_stack(), error_stack)
```

Comment:
Would be easier to understand if the `context` argument here is named `exception` or `error`, I thought this is a `v8::Context` at first

Reply:
Also the one in `CaptureCryptoStatus`

Reply:
Once #16567 lands I'll be refactoring this one a bit. Will make that change then.